README.md

```diff
@@ -41,7 +42,11 @@ addon | version | summary
 [partner_auto_salesman](partner_auto_salesman/) | 8.0.1.0.0 (unported) | Partner auto salesman
 [partner_contact_address_detailed](partner_contact_address_detailed/) | 8.0.1.0.0 (unported) | All address data in summarized contact form
 [partner_contact_birthdate](partner_contact_birthdate/) | 8.0.1.0.0 (unported) | Contact's birthdate
+<<<<<<< 3398e80890c1e14aa3131baad1a5bf6f9ed6a96b
```

Comment:
This has not been correctly rebase. Please remove these changes. As stated, they are done automatically after the merge by an script.